src/main/java/michal/kruczala/project/ptt/ParserPTT.java

```diff
@@ -11,17 +14,27 @@
 public class ParserPTT {
     private static final Logger LOGGER = LoggerFactory.getLogger(ParserPTT.class);
     private static final WebReader webReader = new WebReader();
+    private static final LookForCompNumbers lookForCompNumbersObject = new LookForCompNumbers();
```

Comment:
Jeżeli chodzi o nazwę klasy. Takie klasy, które zajmują się wybieraniem czegoś z tekstu nazywami Parserami, czyli przykładowa nazwa dla tej klasy to: CompetitionNumberParser

Reply:
tak szef!